src/assets/certificates/server.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Please don't push private keys to GH. Creating a security risk.

Reply:
you are not reviewing the latest commit.